docs/guides/key-manager/give-permissions.md

```diff
@@ -128,43 +128,77 @@ The private key can be obtained depending on how you created your Universal Prof
 - UP created with the _lsp-factory.js_: this is the private key given in the `controllerAddresses` field in the method [`lspFactory.UniversalProfile.deploy(...)`](../../tools/lsp-factoryjs/classes/universal-profile#deploy)
 - UP created via the Browser extension: click on the _Settings_ icon (top right) > and _Export Private Key_
 
+<Tabs>
+  
+  <TabItem value="web3js" label="web3.js">
 
 ```javascript title="Load account from a private key"
+import UniversalProfile from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
+import { ERC725 } from '@erc725/erc725.js';
+import LSP6Schema from '@erc725/erc725.js/schemas/LSP6KeyManager.json';
 import Web3 from 'web3';
+
 const web3 = new Web3('https://rpc.l16.lukso.network');
+const myUniversalProfileAddress = '0x...';
 
 const PRIVATE_KEY = '0x...'; // your EOA private key (previously created)
-
 const myEOA = web3.eth.accounts.wallet.add(PRIVATE_KEY);
 ```
 
-### 3.2 - Create contract instance
+  </TabItem>
 
-The next steps is to create the web3 instance of our smart contracts to interact with them. The contract ABIs are available in the @lukso/lsp-smart-contracts npm package.
+  <TabItem value="ethersjs" label="ethers.js">
 
-You will need the address of your Universal Profile deployed on L16.
+```javascript title="Load account from a private key"
+import UniversalProfile from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
```

Comment:
same comment as above.
Good to add the etherjs as an example.

I would even suggest to put etherjs first, as it is more used.